packages/codegen-ui-react/lib/forms/form-renderer-helper/validation.ts

```diff
@@ -40,7 +40,7 @@ export const createValidationExpression = (validationRules: FieldValidationConfi
         ),
       );
     }
-    if ('numValues' in rule) {
+    if ('numValues' in rule && rule.numValues) {
```

Comment:
We should infer the validation rule type based on the `type` field, not on if thos property exists.

Reply:
Done